### PR TITLE
added v to versions

### PR DESCRIPTION
**Pull request by exu**
For fixing deps in executors

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [x] added a test

### CODE REVIEW

File: pkg/version/version_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	t.Run("test get newest version with v prefix", func(t *testing.T) {
+		versions := []string{
+			"v0.1.1",
+			"0.3.2",
+			"v0.3.3",
+			"0.2.2",
+		}
+
+		newest := GetNewest(versions)
+		assert.Equal(t, "0.3.3", newest)
```
Comment (nicufk) [inline pkg/version/version_test.go:20]: Isn't it supposed to test version with v prefix?

Comment (exu) [reply pkg/version/version_test.go:20]: yes it is look at top of change - generally semver package strips v in version string - but is able to read and process it (it's behaving like some kind of prefix and is often omitted)

Comment (exu) [reply pkg/version/version_test.go:20]: @nicufk

Comment (nicufk) [reply pkg/version/version_test.go:20]: 👍

**Review by nicufk — APPROVED** [review]

**Review by jasmingacic — APPROVED** [review]

**Pull request merged**